blaserunning-and-shame/README.md

```diff
@@ -30,15 +30,27 @@ Shame is a uniquely exciting part of blaseball's rules: #watchparty on the offic
 
 ## Methodology and results
 
-Over a sample of 982 game logs, we looked for stolen bases, re-scored games, and determined whether those games ended in shame.
+Our sample consists of 982 game logs where the last out is present.
+Logs start in the middle of Season 2, and a number of games have incomplete logs due to various issues.
+
+On that sample, we looked for stolen bases, re-scored games, and determined whether those games ended in shame.
```

Comment:
consider rephrasing: "In the sample, we searched for stolen bases and re-scored games, then determine whether those games ended in shame."